tests/api2/activedirectory.py

```diff
@@ -291,134 +277,94 @@ def test_23_creating_a_smb_share_on_smb_path(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     results = PUT("/service/id/cifs/", {"enable": True})
     assert results.status_code == 200, results.text
 
 
-def test_28_checking_to_see_if_clif_service_is_enabled(request):
+def test_26_checking_to_see_if_clif_service_is_enabled(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     results = GET("/service?service=cifs")
     assert results.json()[0]["enable"] is True, results.text
 
 
-def test_29_starting_cifs_service(request):
+def test_27_starting_cifs_service(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     payload = {"service": "cifs"}
     results = POST("/service/restart/", payload)
     assert results.status_code == 200, results.text
     sleep(1)
 
 
-def test_30_checking_to_see_if_nfs_service_is_running(request):
+def test_28_checking_to_see_if_cifs_service_is_running(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
     results = GET("/service?service=cifs")
     assert results.json()[0]["state"] == "RUNNING", results.text
 
 
-@bsd_host_cfg
-def test_31_creating_smb_mountpoint(request):
-    depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    results = SSH_TEST('mkdir -p "%s" && sync' % MOUNTPOINT,
-                       BSD_USERNAME, BSD_PASSWORD, BSD_HOST)
-    assert results['result'] is True, results['output']
-
-
-@bsd_host_cfg
-def test_32_store_AD_credentials_in_a_file_for_mount_smbfs(request):
-    depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    cmd = f'echo "[{ip}:{ADUSERNAME.upper()}]" > ~/.nsmbrc && '
-    cmd += f'echo "password={ADPASSWORD}" >> ~/.nsmbrc'
-    results = SSH_TEST(cmd, BSD_USERNAME, BSD_PASSWORD, BSD_HOST)
-    assert results['result'] is True, results['output']
-
-
-@bsd_host_cfg
-def test_33_mounting_SMB(request):
-    depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    cmd = f'mount_smbfs -N -I {ip} -W AD01 -U {ADUSERNAME} ' \
-        f'"//{ADUSERNAME}@{ip}/{SMB_NAME}" "{MOUNTPOINT}"'
-    results = SSH_TEST(cmd, BSD_USERNAME, BSD_PASSWORD, BSD_HOST)
-    assert results['result'] is True, results['output']
-
-
-@bsd_host_cfg
-def test_34_creating_SMB_file(request):
-    depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    results = SSH_TEST('touch "%s/testfile"' % MOUNTPOINT,
-                       BSD_USERNAME, BSD_PASSWORD, BSD_HOST)
-    assert results['result'] is True, results['output']
-
-
-@bsd_host_cfg
-def test_35_moving_SMB_file(request):
+def test_29_create_a_file_and_put_on_the_active_directory_share(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    cmd = 'mv "%s/testfile" "%s/testfile2"' % (MOUNTPOINT, MOUNTPOINT)
-    results = SSH_TEST(cmd, BSD_USERNAME, BSD_PASSWORD, BSD_HOST)
+    cmd_test('touch testfile.txt')
+    print()
+    command = fr'smbclient //{ip}/{smb_name} -U {cmd_ad_user}%{ADPASSWORD}' \
+        r' -m NT1 -c "put testfile.txt testfile.txt"'
```

Comment:
Out of curiosity, is there a particular reason we're using SMB1 for this test? If not it's better to just leave default protocol in place, and explicitly test SMB1 vs SMB3 in other SMB protocol-specific tests e.g. test_425.

Reply:
If you think it is too much testing for SMB1, I can remove that.

Reply:
Not a matter of too much testing. Just a philosophical point. :) 
We should probably stick to our default settings unless we're explicitly testing a non-default configuration.

Reply:
Done